### PR TITLE
Disable second external network

### PR DESCRIPTION
**Pull request by ntymtsiv**

### CODE REVIEW

**Comment by ntymtsiv** [comment]
In configuration.ini dst_storage and src_storage should be nfs instead of ceph

**Comment by plomakin** [comment]
+1

**Comment by plomakin** [comment]
looks good

File: devlab/tests/config.py [hunk 1]
```diff
@@ -157,14 +157,14 @@
...
-              {'start': '192.168.7.10', 'end': '192.168.7.192'}]
-          }]
+     #   },
+     # {'name': 'second_shared_net', 'admin_state_up': True, 'shared': True,
+     #     'router:external': True,
+     #     'subnets': [
+     #     {'cidr': '192.168.7.0/24', 'ip_version': 4, 'name': 'shared_subnet',
+     #          'routers_to_connect': [], 'allocation_pools': [
+     #              {'start': '192.168.7.10', 'end': '192.168.7.192'}]
+     #          }]
```
Comment (roman-verchikov) [inline devlab/tests/config.py]: Please either remove or leave, don't comment

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**